src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/message/ReclusterService.kt

```diff
@@ -128,9 +128,6 @@ class ReclusterService(
     // Step 1: Mark the source key as merged
     sourcePersonKey.mergedTo = targetPersonKey.id
     sourcePersonKey.status = UUIDStatusType.RECLUSTER_MERGE
-
-    // Step 2: Detach source entity from its original personKey
-    sourcePersonKey.personEntities.remove(sourcePersonEntity)
     personKeyRepository.save(sourcePersonKey)
 
     // Step 3: Attach the source entity to the target personKey
```

Comment:
not mad on the comments but this is no longer step 3 (which is a good illustration of what I don't like about comments - they get out of date quickly)